OsmAnd/src/net/osmand/plus/settings/backend/backup/items/SettingsItem.java

```diff
@@ -113,8 +113,11 @@ public void setLastModifiedTime(long lastModified) {
 	public abstract long getEstimatedSize();
 
 	public boolean applyFileName(@NonNull String fileName) {
+		// Case: this.fileName could be a folder so all remote files will be collected for it ?
+		// + Subfolder check correct
+		// - Where is type prefix ? filename same for different types
 		String n = getFileName();
-		return n != null && (n.endsWith(fileName) || fileName.startsWith(n + File.separator));
```

Comment:
This is incorrect ! 
- https://github.com/osmandapp/OsmAnd-iOS/commit/19aebad81653a504ba436df58c8f97a8f36a52aa
- + Type prefix should be checked